### ISSUE TITLE
List links should be underlined

### ISSUE THREAD

**Issue by Fupete** [italia/design-ui-kit#119]
### Does a discussion already exist on the topic you are interested in, or on a similar topic?

- [X] I verified and it doesn't exist

### What

We need to fix here also this new Bootstrap Italia issue: https://github.com/italia/bootstrap-italia/issues/1077

Aggiornato da Fupete 17/9/2024:
Aggiungo alla presente anche il problema che il "Link attivo" ha lo stesso colore del testo attualmente 🤔 

### Perché
Aggiornato da Fupete 17/9/2024:
Visivamente accessibile. Comprensibile. 

### Contesto
Aggiornato da Fupete 17/9/2024:
Verifiche di accessibilità sui componenti.

⚠️ Sarà da aggiornare lo status delle verifiche nella [board census ](https://github.com/orgs/italia/projects/32/views/1)alla risoluzione.

### Altro
Aggiornato da Fupete 17/9/2024:

Problema presente anche in UI Kit Italia, React kit e Angular kit. 
Qui la issue UI Kit Italia: https://github.com/italia/design-ui-kit/issues/119

**Comment by zetareticoli** [comment]
@Fupete the `active` state is useless because the general list isn't a menu (we have the `Sidebar` component for that). 

I suggest you remove it from examples (doing it on BS Italia). Do you agree?
